src/main/sensors/battery.c

```diff
@@ -329,10 +338,10 @@ static void batteryUpdateConsumptionState(void)
 void batteryUpdateCurrentMeter(timeUs_t currentTimeUs)
 {
     UNUSED(currentTimeUs);
-    if (batteryCellCount == 0) {
-        currentMeterReset(&currentMeter);
-        return;
-    }
+//    if (batteryCellCount == 0) {
+//        currentMeterReset(&currentMeter);
+//        return;
+//    }
```

Comment:
this is disabled because the current sensors currently require that a battery is connected/detected, I'm not sure that approach is correct because it's possible to have no voltage monitoring but some current monitoring.